src/plugins/intel_gpu/src/graph/impls/ocl/arg_max_min.cpp

```diff
@@ -60,22 +60,22 @@ struct arg_max_min_impl : typed_primitive_impl_ocl<arg_max_min> {
     }
 
 public:
-    static std::unique_ptr<primitive_impl> create(const arg_max_min_node& arg, const kernel_impl_params& impl_param) {
+    static kernel_params_t get_kernel_params(const kernel_impl_params& impl_param) {
         const auto& primitive = impl_param.typed_desc<arg_max_min>();
         const auto& axis = primitive->axis;
         const auto& top_k = primitive->top_k;
         const auto& mode = primitive->mode;
         const auto& sort_type = primitive->sort;
         const auto& values_first = primitive->values_first;
-        const auto& outputs_num = arg.get_output_nums();  // second output passed as input for TOP_K layer
+        const auto& outputs_num = (primitive->input_size() == 3 ? 2 : primitive->output_size());  // second output passed as input for TOP_K layer
```

Comment:
Why this is changed?

Reply:
because I want to remove arg_max_min_node arg from get_kernel_params's input parameters.